README.md

```diff
@@ -282,118 +551,10 @@ ie.
     sudo mscs send alpha say Hello world!
 
 
-### Import Existing Worlds
-
-You just need to create a new directory in the worlds folder for the world you wish to import.
-Suppose the world you wish to import is called `alpha`, you would create a new folder in
-`/opt/mscs/worlds`, then copy the data files over to that directory.
-
-If the directory containing the world `alpha` you wish to import looks like this:
-
-    $ ls
-    alpha
-    banned-ips.txt
-    banned-players.txt
-    crash-reports
-    logs
-    ops.txt
-    server.properties
-    white-list.txt
-
-You can just copy your world into the worlds directory:
-
-    mkdir /opt/mscs/worlds/alpha
-    cp -R * /opt/mscs/worlds/alpha
-
-Make sure you check `server-port` and `query.port` in `server.properties` to make sure it does not overlap with other servers created by the MSCS script. Also ensure that `enable-query` is set to `true`.  If you do not have `enable-query` and a `query.port` set, you will not be able to check the status of the world with the script.
-
```

Comment:
The directions I wrote here are correct.  We can expand upon them, and talk about how to rename a world the hard way (since I haven't written any code for an easy way yet).

This is untested, but something like this would work to rename `alpha` to `vanillaMC`:
    mkdir /opt/mscs/worlds/vanillaMC
    cp -R * /opt/mscs/worlds/vanillaMC
    mv /opt/mscs/worlds/vanillaMC/alpha /opt/mscs/worlds/vanillaMC/vanillaMC
    mscs create vanillaMC 25565